libs/web/src/components/common/Footer/Footer.tsx

```
@@ -1,66 +1,54 @@
-            <a href={DiscordLink}>Support</a>
-          </Button>
-          <Button
-            asChild
-            variant="link"
-            className="text-content-tertiary hover:text-content-primary hover:no-underline"
+            Support
+          </a>
+
+          <a
+            href="https://docs.mira.ly/resources/security-audits"
+            target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)